controls/radbuttons/styles-and-templates/styling-buttonchrome.md

```diff
@@ -10,7 +10,23 @@ position: 2
 
 # Styling the ButtonChrome
 
-The __ButtonChrome__ control was part of the template of the __RadButtons__. It was responsible for the visual appearance of the button. If you're using an older version of the RadControls, this article will help you style the __ButtonChrome__ control. It can be styled by creating an appropriate __Style__ and setting it to the __Style__ property of the control. 
+__ButtonChrome__ is a Control that has no input functionality. It was part of the template of the __RadButtons__. It was responsible for the visual appearance of the button. If you're using an older version of the RadControls, this article will help you style the __ButtonChrome__ control. It can be styled by creating an appropriate __Style__ and setting it to the __Style__ property of the control. 
+
+## Properties
+
+The Button chrome’s appearance is controlled by several Boolean typed properties:
+* __RenderMouseOver__: render as MouseOvered button;

```

Comment:
The first two items say "render" and the others are "renders". Add "s" at the end for the first as well.